src/main/java/org/squiddev/cobalt/function/UnwindableCallable.java

```diff
@@ -0,0 +1,10 @@
+package org.squiddev.cobalt.function;
+
+import org.squiddev.cobalt.LuaError;
+import org.squiddev.cobalt.UnwindThrowable;
+import org.squiddev.cobalt.debug.DebugFrame;
+
+@FunctionalInterface
+public interface UnwindableCallable<R> {
```

Comment:
I'd be inclined to make this no longer generic, and just return `EvalCont`. Should mean we can get rid of that unchecked suppression, and a load of `CHECKCAST` operations.

Reply:
That's a very good point, I'll fix it shortly.